src/extension.ts

```diff
@@ -23,6 +23,25 @@ export async function activate(context: vscode.ExtensionContext): Promise<void>
   reporter = new TelemetryReporter(context.extension.id, manifest.version, manifest.appInsightsKey);
   context.subscriptions.push(reporter);
 
+  const stable = vscode.extensions.getExtension('hashicorp.terraform');
+  const preview = vscode.extensions.getExtension('hashicorp.terraform-preview');
+
+  if (context.extension.id === 'hashicorp.terraform-preview') {
+    if (stable !== undefined) {
```

Comment:
Is it worth adding a comment here to say that `getExtension` returns `undefined` when the extension ID is either not installed or disabled? Just to clarify...